test/csuite/timestamp_abort/main.c

```diff
@@ -230,9 +230,11 @@ thread_ckpt_run(void *arg)
         fflush(stdout);
         /*
          * Create the checkpoint file so that the parent process knows at least one checkpoint has
-         * finished and can start its timer.
+         * finished and can start its timer. If running with timestamps, wait until stable timestamp
+         * has moved past WT_TS_NONE to give a chance to writer threads to add something to the
+         * database.
```

Comment:
I suggest a minor rewording of the new text to be:
"If running with timestamps, wait until the stable timestamp moves past WT_TS_NONE to give writer threads a chance to add something to the database."

Reply:
Fixed.